app/services/order_row_importer.rb

```diff
@@ -152,6 +154,14 @@ def add_product_to_order
     end
   end
 
+  def to_add_attributes
+    { note: field(:notes), account:, reference_id: field(:reference_id), project_id: field(:project_id) }.merge(custom_attributes)
```

Comment:
Should this include `project_id: field(:project_id)`? From what I understand, we use `merge(custom_attributes)` and have the `custom_attributes` from the extension add that field. Am I missing something?

Reply:
@LeticiaErrandonea good catch - project id should be removed here